XenAdmin/Wizards/RollingUpgradeWizard/RollingUpgradeWizardPrecheckPage.cs

```diff
@@ -165,6 +165,21 @@ protected override List<CheckGroup> GenerateChecks(Pool_patch patch)
                 groups.Add(new CheckGroup(Messages.CHECKING_XENCENTER_VERSION, new List<Check> {new XenCenterVersionCheck(null)}));
             }
 
+            //Checking PV guests - for hosts that have any PV guests and warn the user before the update/upgrade.

```

Comment:
This check should move after the liveness checks; I think a better place would be towards the end after the post-upgrade memory check.